site/docs/master/aws-config.md

```diff
@@ -18,9 +18,10 @@ Velero. The tarballs for each release contain the `velero` command-line client.
 of the Velero repository is under active development and is not guaranteed to be stable!_
 
 2. Extract the tarball:
-```bash
+
+    ```bash
 tar -xvf <RELEASE-TARBALL-NAME>.tar.gz -C /dir/to/extract/to
```

Comment:
what's up with this indenting style, where the ```bash is indented but the content isn't? It doesn't render properly in GitHub (see https://github.com/heptio/velero/blob/4aca7da1da45e05b30459fe2525a0be35cb66705/site/docs/master/aws-config.md). I *think* indenting the entire thing should work, no?

Reply:
That was to make sure it got rendered correctly on the docs site. There was a mix of different styles implemented already in the existing docs, so I picked the most prevalent one and made it work on the site. I see what you're saying about the incorrect render on standard GitHub Markdown. I'll make sure it renders correctly in both places.

Reply:
Indentation should now be fixed.